modules/javafx.controls/src/test/java/test/javafx/scene/control/ListViewKeyInputTest.java

```diff
@@ -1841,6 +1841,7 @@ private boolean isAnchor(int index) {
             listView.getItems().add("Row " + i);
         }
         listView.setPrefHeight(130); // roughly room for four rows
+        Toolkit.getToolkit().firePulse();
```

Comment:
Why is this needed here?

Reply:
Without this line the test fails in the line 1865 with the message 'expected:<7> but was:<8>'.
I think this is because in the line 1850 'listView.scrollTo(99)' is executed, which now does not set the position to 1.

Reply:
That is very well possible indeed. Since we estimate the size of the different cells, it is hard to deterministically put values on the counts. Therefor, I changed a number of tests from `equals` to `<` . I would personally suggest not adding the `pulse` here, but rather change the test in line 1865 with `... < 10 ` to make sure that there is no abnormal amount of processing. (and maybe also `> 0 ` to make sure that the updateItem is at least counted once).